espnet2/layers/sinc_conv.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+#  2020, Technische Universität München;  Ludwig Kürzinger
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+"""Sinc convolutions."""
+import torch
+from typing import Union
+
+CONSTANT_PI = 3.141592653589793
```

Comment:
I'm curious why you represent \pi as follows.
I just thought that you can use `math.pi`.

Reply:
Correct!
In a previous version, we used `np.pi` and there is not yet a `torch.pi`.
My thought here was that setting the constant is faster than importing the numpy library only for that one number....
Also, one dependency less.

Reply:
Please use math.pi.
Not faster, it's always same.

Reply:
OK